.github/workflows/deploy-appengine-creds-it.yml

```diff
@@ -118,21 +153,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: create app
-      run: |-
-        cat <<EOF > ${{ github.workspace }}/app.yaml
-        service: "${{ github.job }}-${{ github.run_number }}"
-        runtime: "nodejs10"
-        EOF
-
-        cat <<EOF > ${{ github.workspace }}/server.js
-        const http = require('http');
-        const server = http.createServer(function (req, res) {
-          res.writeHead(200)
-          res.end('Hello world!');
-        });
-        server.listen(process.env.PORT || 8080);
-        EOF
+    - name: Update app.yaml
+      run: |-
+        echo "service: ${{ github.job }}-${{ github.run_number }}" >> ${{ github.workspace }}/example-app/app.yaml
+        cat ${{ github.workspace }}/example-app/app.yaml
```

Comment:
Are these needed?

Reply:
This creates a field in the app.yaml for GAE to create a non-default service. It must be a named service in order for the tests to clean up the resource. The initial set up created and `index.js` file to deploy instead of just pointing to our example app.

Reply:
I was wondering about the cat/ls statements after the replace

Reply:
Oops, will remove debugging. Thanks for catching